### DISCUSSION TITLE
if  you accept my PR , but my name doesn't appear in Leaderboard , is that means my program is running for more than 15 mins ? or I should wait

### DISCUSSION THREAD

**Discussion by sohaibMan** [geeksblabla/blanat#59] [Q&A] [answered]
thank you

**Comment by aboullaite** [comment, marked as answer]
Either it timed out or the result wasn't correct

**Reply by dauom** [reply]
@aboullaite  is there any chance you guys can report back in the PR which result was it (compilation error, wrong answer, time limit, presentation error) after you merge it? that would help us to fix the code.

**Reply by aboullaite** [reply]
We can't commit to that tbh. You can generate the full dataset on your end now with the python script and try your solution before submitting it.

**Reply by dauom** [reply]
I did that and cross-checked the result with the top 2 solutions in the leaderboard but still don't see the problem (same output).

**Reply by aboullaite** [reply]
@dauom Check the updated leaderboard. Congrats (for now :D)
